packages/engine/src/args.rs

```diff
@@ -54,6 +54,23 @@ pub struct Args {
     /// The size at which the engine aims to split a group of agents
     #[clap(long, default_value = "100000", env = "ENGINE_TARGET_MAX_GROUP_SIZE")]
     pub target_max_group_size: usize,
+
+    /// Allows a JavaScript runner to grow to some initial size in megabytes before triggering
```

Comment:
If you apply my comments for the other args struct, please remember to copy them over to this